File: Modul6/Python/PRAK601-2310817210020-Muhammad_Hasbiyan_Rusyadi.py
a, b = map(int,input().split())
arr = []
for _ in range(a):
    row = list(map(int,input().split()))
    arr.extend(row)
for y in range(a):
    for z in range(b):
        print(arr[y * b + z], end=" ")
    print()

File: Modul6/C/PRAK601-2310817210020-Muhammad_Hasbiyan_Rusyadi.c
#include<stdio.h>
int main() {
    int a,b;
    scanf("%d %d", &a,&b);
    int arr[a][b];
        for(int y = 0 ; y < a ; y++) {
            for(int z = 0 ; z < b ; z++){
                scanf("%d", &arr[y][z]);}}
        for(int y = 0 ; y < a ; y++) {
            for(int z = 0 ; z < b ; z++){
                printf("%d ", arr[y][z]);}
            printf("\n");}
        return 0;}